.clang-format

```diff
@@ -40,6 +40,11 @@ IndentCaseLabels: true
 PointerAlignment: Left
 DerivePointerAlignment: true
 
+# Move public and private in by a half-indentation.  This makes
+# diffs and Github code reviews more readable by letting you
+# see which class the diff snippet is part of.
+AccessModifierOffset: -1
```

Comment:
I kinda wanted it to align with V8 and node.js style, which do just 1.

When you say 2, do you want it more indented than the methods of the class?

Reply:
Much prefer 1 here.

Reply:
So rather than -1, I recommend having 1.

Reply:
To be clear (I'm not sure what the difference between using the value -1 or 1 is here)... I'd like the format to align consistently with Node.js and v8 so that the result is the same as what Erik has done here...

```
class Foo {
 // one character indent for public:
 public:
  Foo() = default;
}
```

Reply:
My understanding is making it -1 doesn't affect the ancestors of accessors. If we made it 1, we'd see a much bigger git diff.

Reply:
If you make it 1 instead of -1 then you get

```
class Foo {
   public:
  Foo() = default;
  
   private:
  int field;
}
```